src/libraries/System.Numerics.Vectors/tests/Matrix4x4LeftHandTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Numerics.Tests
+{
+    public class Matrix4x4LeftHandTests : Matrix4x4OrientationAwareTests
+    {
+        [Fact]
+        public void Matrix4x4CreateLookAtLeftHandedTest()
+        {
+            Vector3 cameraPosition = new Vector3(10.0f, 20.0f, 30.0f);
+            Vector3 cameraTarget = new Vector3(3.0f, 2.0f, -4.0f);
+            Vector3 cameraUpVector = new Vector3(0.0f, 1.0f, 0.0f);
+
+            Matrix4x4 expected = new Matrix4x4();
+            expected.M11 = -0.979457f;
+            expected.M12 = -0.0928268f;
+            expected.M13 = -0.179017f;
+
+            expected.M21 = +0.0f;
+            expected.M22 = +0.887748f;
+            expected.M23 = -0.460329f;
+
+            expected.M31 = +0.201653f;
+            expected.M32 = -0.450873f;
+            expected.M33 = -0.869511f;
+
+            expected.M41 = +3.74498f;
+            expected.M42 = -3.30051f;
+            expected.M43 = +37.0821f;
+            expected.M44 = +1.0f;
+
+            Matrix4x4 actual = CreateLookAt(cameraPosition, cameraTarget, cameraUpVector);
+            Assert.True(MathHelper.Equal(expected, actual), $"{CreateLookAtMethodName} did not return the expected value.");
```

Comment:
nit: We have existing `AssertExtensions.Equal(expected, actual, allowedVariance);` helper that will provide better diagnostic messages in the case of failure.

Reply:
I've updated this and used `1e-5` for the variance (matches `MathHelper.Equals`). I think we can can use tighter errors in some cases where we have numbers with abs < 1 and having 6 sigfigs, but I'll just leave it for now.